src/Application/SSW.Rewards.Application.csproj

```diff
@@ -26,7 +26,7 @@
 
     <ItemGroup>
```

Comment:
I don't this we need this `ItemGroup`. 
AFAIN, `Folder` is used when you need to generate classes and put them into a non-existing folder.
[Docs](https://learn.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2022#folder)

Reply:
Fixed